src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Dynamic Sampling
+sidebar_order: 6
+description: "Learn about the different methods for limiting the amount of data generated by your project."
+---
+
+
+TODO -  Should we use Dynamic Sampling or something like: "Centralized Sampling" or "Server Side Sampling" (I think
+that, from a user's perspective, "Dynamic Sampling" doesn't convey much information).
+
+
+
+# Why Dynamic Sampling
+
+Sentry SDKs are able to sample events by setting `traces_sample_rate` or `traces_sampler` during SDK initialisation
+(TODO add link to .../PerformanceMonitoring/SamplingTransactions).
```

Comment:
Note: I'll be moving this into the Configuration section for the SDKs to align with the Filtering content. But feel free to move forward. I'll take care of fixing any links I break in the process :)